ruby/vampires.rb

```diff
@@ -1,82 +1,115 @@
+employee_count = gets.to_i
 
-print "What is your name?"
-employee_name = gets.chomp
-case employee_name
-when "Drake Cula"
-	vamp_handle = true
-when "Tu Fang"
-	vamp_handle = true
-else 
-	vamp_handle = false
-end
+index = 0
 
-print "How old are you?" 
-age = gets.to_i
+while index < employee_count
 
-print "What year were you born?"
-birth_year = gets.to_i
-current_year = 2016
+	puts "What is your name?"
+	employee_name = gets.chomp
+	case employee_name
+	when "Drake Cula"
+		vamp_handle = true
+	when "Tu Fang"
+		vamp_handle = true
+	else 
+		vamp_handle = false
+	end
 
-if current_year - birth_year === age
-	real_age = true
-else
-	real_age = false
-end
+	puts "How old are you?" 
+	age = gets.to_i
 
-print "Our company cafeteria serves garlic bread. Should we order some for you?"
-garlic = gets.chomp
-case garlic
-when "yes"
-	likes_garlic = true
-when "no"
-	likes_garlic = false
-end
-		
+	puts "What year were you born?"
+	birth_year = gets.to_i
+	current_year = 2016
 
-print "Would you like to enroll in the company’s health insurance?"
-insurance = gets.chomp
-case insurance
-when "yes"
-	mortal = true
-when "no"
-	mortal = false
-end
+	if current_year - birth_year === age
+		real_age = true
+	else
+		real_age = false
+	end
 
-# print "#{employee_name}, #{age}, #{birth_year}, #{garlic}, #{insurance}."
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+	garlic = gets.chomp
+	case garlic
+	when "yes"
+		likes_garlic = true
+	when "no"
+		likes_garlic = false
+	end
+			
 
+	puts "Would you like to enroll in the company’s health insurance?"
+	insurance = gets.chomp
+	case insurance
+	when "yes"
+		mortal = true
+	when "no"
+		mortal = false
+	end
 
+	# print "#{employee_name}, #{age}, #{birth_year}, #{garlic}, #{insurance}."
 
+	puts "If you have any allergies, please list the first"
+		employee_allergy = gets.chomp
 
-if real_age && (likes_garlic || mortal)
-	probably_not = true
-elsif (real_age === false) && (likes_garlic === false && mortal === true)
-	probably_is = true
-elsif (real_age === false) && (likes_garlic === true && mortal === false)
-	probably_is = true
-elsif (real_age === false) && (likes_garlic === false && mortal === false)
-	almost_certain = true
-else  
-	inconclusive = true 
-end
+	until employee_allergy === "done" || employee_allergy === "sunshine"	
+		puts "Please list your next allergy. If none, type 'done'."
+		employee_allergy = gets.chomp
+	end
 
-if probably_not && (vamp_handle === false)
-	puts "Probably not a vampire."
-elsif probably_is && (vamp_handle === false)
-	puts "Probably a vampire."
-elsif almost_certain && (vamp_handle === false)
-	puts "Almost certainly a vampire."
-elsif probably_not && vamp_handle
-	puts "Definitely a vampire."
-elsif probably_is && vamp_handle
-	puts "Definitely a vampire."
-elsif almost_certain && vamp_handle
-	puts "Definitely a vampire."
-else
-	puts "Results inconclusive."
+	case employee_allergy
+	when "sunshine"

```

Comment:
Avoid `case` statements with most control flow.  Especially if we are looking for only 1 or 2 unique conditions.